api/api-doc/storage_service.json

```diff
@@ -797,6 +797,21 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/cleanup",
```

Comment:
Why not reuse `cleanup_all` and fallback inside the handler?

Reply:
the summary of it reads "Trigger a global cluster cleanup through the topology coordinator". while this newly introduced API performs the cleanup locally without using RPC or group0. do we want to preserve the behavior and the summary of `cleanup_all` ?

Reply:
> @gleb-cloudius the summary of it reads "Trigger a global cluster cleanup through the topology coordinator". while this newly introduced API performs the cleanup locally without using RPC or group0. do we want to preserve the behavior and the summary of `cleanup_all` ?

We want it to cleanup through the coordinator if available and without otherwise. Currently the fallback in the nodetool because it already had the code to go over all keyspaces to cleanup everything. If you add this code to Scylla itself lets move fallback there too.

Reply:
@gleb-cloudius i understand your intention. my concern is only about the semantics of "cleanup_all" and its connection with  "the topology coordinator", if i am allowed to change it to  "Trigger a global cluster cleanup", i would be happy to move the fallback to the server side instead of adding yet another API.

Reply:
thanks! will do.